app/models/log_event.rb

```diff
@@ -0,0 +1,6 @@
+class LogEvent < ActiveRecord::Base
+
+  belongs_to :user
```

Comment:
The `user_id` still references a user in the ideal case, even if we aren't sure enough that it's always a real user enough to force the database to enforce it.